ui/hooks/src/use-posts.ts

```diff
@@ -211,8 +211,7 @@ const getPost = async (postID: string, loggedUser?: string) => {
 
 /**
  * Hook to get data for a specific post
- * @param postId - id of the post
- * @param loggedUser - pubKey of the currently logged in user
+ * @param param0 - payload containg the id of the post and textile public key of the logged in user
```

Comment:
param0 ? maybe remove this @param?

Reply:
yes, because it was de-structured in place. I'll remove the @param